roles/openshift_health_checker/openshift_checks/logging/fluentd_config.py

```diff
@@ -0,0 +1,144 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+from openshift_checks.ovs_version import OvsVersion
+
+
+class FluentdConfig(LoggingCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
```

Comment:
`fluentd` would run on the first master only, while `fluentd_config` could theoretically run on all the nodes, but I'm forgetting now how the impl goes and maybe it doesn't. If is_active is different it makes sense to keep separate...